### DISCUSSION TITLE
Keda deployment machine selector

### DISCUSSION THREAD

**Discussion by paoloalba** [kedacore/keda#844] [General]
Does the deployment work with any kind of machine? If only linux ones are supported, a selector should be added.

**Comment by ahmelsayed** [comment]
We only build for Linux, so yes I think we should set a selector for that.

I don't think there is anything in the code that should prevent it from working on Windows if needed, but I never tested that myself.

**Reply by paoloalba** [reply]
I did try to deploy 1.4.1 on my cluster, and I got pull errors when keda pods got assigned to windows nodes. I solved it indeed manually assigning a selector.

**Reply by zroubalik** [reply]
@paoloalba would be great if you can send a PR. I suppose we would need to build windows image to be able to schedule KEDA on Windows.
